src/test/scala/org/tensorframes/impl/TestUtils.scala

```diff
@@ -17,13 +17,12 @@
 package org.tensorframes.impl
 
 import java.nio.file.{Files, Paths => JPaths}
-import scala.collection.JavaConverters._
 
-import org.tensorflow.{Graph => TFGraph, Session => TFSession, Output => TFOut, Tensor}
 import org.tensorflow.framework.GraphDef
-
+import org.tensorflow.{Tensor, Graph => TFGraph, Output => TFOut, Session => TFSession}
 import org.tensorframes.ShapeDescription
-import org.tensorframes.dsl.Implicits._
+
+import scala.collection.JavaConverters._
```

Comment:
Nit: Scala import should appear after the Java import but before the TensorFlow import, see [relevant section of style guide](https://github.com/databricks/scala-style-guide#imports)